views/App.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+import { withStyles } from '@material-ui/core/styles'
+
+import withRoot from './hoc/withRoot'
+import { AuthContext } from './contexts/AuthContext'
+import Router from './routes'
+import { styles } from './styles'
+
+import 'react-virtualized/styles.css'
+
+const App = (props) => {
+  const [user, setUser] = useState(null)
+
+  return (
+    <AuthContext.Provider value={[user, setUser]}>
+      <Router {...props} user={user} setUser={setUser} />
```

Comment:
what props are we expecting to pass from the `App` to the `Router` aside from `user` and `setUser`? We also shouldn't need to pass those props since they are available via context.

Reply:
The classes and theme props used by the material components